### PR TITLE
Wfs

### PR DESCRIPTION
**Pull request by YusufMavzer**

### CODE REVIEW

File: src/StreetNameRegistry.Projections.Wfs/StreetNameHelper/StreetNameHelperProjections.cs [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    using NodaTime;
+
+    [ConnectedProjectionName("WFS straatnaam")]
+    [ConnectedProjectionDescription("Projectie die de straatnaam data voor het WFS straatnaamregister voorziet.")]
+    public class StreetNameHelperProjections: ConnectedProjection<WfsContext>
+    {
+
+        public StreetNameHelperProjections()
+        {
+                        When<Envelope<StreetNameWasRegistered>>(async (context, message, ct) =>
```
Comment (ArneD) [inline src/StreetNameRegistry.Projections.Wfs/StreetNameHelper/StreetNameHelperProjections.cs:15]: indenting

File: src/StreetNameRegistry.Projections.Wfs/StreetNameHelper/StreetNameHelperProjections.cs [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+                    message.Message.StreetNameId,
+                    entity =>
+                    {
+                        entity.Status = null;
+                        UpdateVersionTimestamp(entity, message.Message.Provenance.Timestamp);
+                    },
+                    ct);
+            });
+        }
+
```
Comment (ArneD) [inline src/StreetNameRegistry.Projections.Wfs/StreetNameHelper/StreetNameHelperProjections.cs:269]: Don't forget our new event `WasProposedV2` :)

**Review by ArneD — CHANGES_REQUESTED** [review]
Very good so far, only 3 small remarks!

File: paket.dependencies [hunk 3]
```diff
@@ -13,6 +13,7 @@ nuget Microsoft.Extensions.Http.Polly 5.0.1
 // For more healtchecks, look at https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks
 nuget AspNetCore.HealthChecks.SqlServer 5.0.3
 
+nuget Microsoft.EntityFrameworkCore.SqlServer.NetTopologySuite 5.0.6
```
[the remark this answers is not in the document]
Comment (YusufMavzer) [reply paket.dependencies:16]: That right. I'll take it out

File: src/StreetNameRegistry.Projections.Wfs/StreetNameHelper/StreetNameHelperProjections.cs (hunk 2 above)
Comment (YusufMavzer) [reply src/StreetNameRegistry.Projections.Wfs/StreetNameHelper/StreetNameHelperProjections.cs:269]: True almost forgot

**Pull request merged**